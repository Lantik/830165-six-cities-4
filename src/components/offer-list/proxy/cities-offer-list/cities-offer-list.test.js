import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import CitiesOfferList from './cities-offer-list.jsx';

const mock = {
  offers: [{}, {}, {}]
};

it(`<CitiesOfferList /> rendered correctly`, () => {
  const {offers} = mock;
  const renderer = new ShallowRenderer();

  renderer.render(<CitiesOfferList
    offers={offers}
    onOfferHeaderClick={() => {}}
  />);
  const tree = renderer.getRenderOutput();

  expect(tree).toMatchSnapshot();
});
