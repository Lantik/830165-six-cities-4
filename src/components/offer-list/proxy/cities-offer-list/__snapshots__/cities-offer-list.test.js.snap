// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CitiesOfferList /> rendered correctly 1`] = `
<OfferList
  className="cities__places-list places__list tabs__content"
  offerComponent={[Function]}
  offers={
    Array [
      Object {},
      Object {},
      Object {},
    ]
  }
  onOfferHeaderClick={[Function]}
/>
`;
